ow_power_system/include/PowerModelHandler.h

```diff
@@ -0,0 +1,113 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// This is the header file for the PowerModelHandler class, which handles
+// the simulation of a single model within a PowerSystemNode. It stores the
```

Comment:
I've not heard of a "power model" as a singular entity.  Is this explained somewhere?  A diagram with the relations would be helpful.

Reply:
Maybe a better name would be ``BatteryModelHandler``? The purpose of the class is to handle the inputs going into whatever battery model GSAP's prognosers are simulating (based on the parameters set in ``prognoser.cfg`` & ``async_prognoser.cfg``). I didn't want to call it a ``PowerSystemCell`` because of the developments with Chetan involving using a different model that represents more than one cell at a time.

Changing the name again shouldn't be too difficult compared to before, I imagine most comments and variables would remain the same from ``PowerModelHandler`` to ``BatteryModelHandler``. That said, if you've got an idea for a different name for the class, I'm all ears.

Reply:
I can't think of a better name without seeing the bigger picture, which would be nicely conveyed in a UML/entity relation diagram like the one found in https://github.com/nasa/ow_simulator/wiki/Autonomy.  Such a diagram would help understand the power system and be a good addition to the user guide.  

Could you create one?  If you don't have an easy diagramming tool, don't spend a lot of time on it.  You could even sketch one on paper and photograph it.

I use the (non-free) Mac tool Omnigraffle and can create diagrams quickly.  It's much faster than Powerpoint or Confluence's Gliffy.

Reply:
Sure thing, I'll get to work on that. I was looking at OW-768 (Diagram the Power System) while waiting on the PR reviews a few weeks back, and it got put on the backburner once they came through.

I didn't find Gliffy to be too cumbersome last time I used it, though I was sticking to the basics. I should be able to draft something up today.